### PR TITLE
migrated to express. change to accept only cid, not urls

### PR DESCRIPTION
**Pull request by femans**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,9 +34,12 @@
     "@tensorflow/tfjs-node": "^3.11.0",
```
Comment (dokterbob) [inline package.json:34]: It seems hapi could be removed here.

File: src/nsfwServerHapi.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
+// eslint-disable-next-line no-unused-expressions,global-require
```
Comment (dokterbob) [inline src/nsfwServerHapi.js:1]: Can't this entire file be removed now?

**Review by dokterbob — CHANGES_REQUESTED** [review]
Test error:   ● mozilla grapefruit jpg › should return a 200 status code

File: src/nsfwServerHapi.js (hunk 2 above)
Comment (femans) [reply src/nsfwServerHapi.js:1]: yes

File: src/nsfwServer.js [hunk 3]
```diff
@@ -1,106 +1,72 @@
...
-    },
+    try {
+      const decodedImage = await tf.node.decodeImage(axiosResponse.data, 3);
+      const classification = await model.classify(decodedImage);
+      decodedImage.dispose();
+
+      console.log(`classification for ${cid}:`, classification);
+      return res.status(200).send({
+        classification: Object.fromEntries(classification.map(
+          (entry) => [entry.className.toLowerCase(), entry.probability],
```
[the remark this answers is not in the document]
Comment (femans) [reply src/nsfwServer.js:60]: i did an eslint fix and pushed it now

**Comment by femans** [comment]
I believe the unit tests are not yet working

**Pull request merged**